extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -68,17 +79,28 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
       true /** callToInitialize */
     );
 
-    this.storeService_.subscribe(StateProperty.RTL_STATE, (rtlState) => {
-      this.onRtlStateUpdate_(rtlState);
-    });
+    this.storeService_.subscribe(StateProperty.RTL_STATE, (rtlState) =>
+      this.onRtlStateUpdate_(rtlState)
+    );
 
     this.storeService_.subscribe(
       StateProperty.PAGE_SIZE,
-      (pageSizeState) => {
-        this.flipTagIfOffscreen_(pageSizeState);
-      },
+      (pageSizeState) => this.flipTagIfOffscreen_(pageSizeState),
       true /** callToInitialize */
     );
+
+    this.storeService_.subscribe(StateProperty.CURRENT_PAGE_ID, (id) =>
+      this.toggleShoppingTagActive_(id)
+    );
+  }
+
+  /**
+   * @param {!string} id
+   * @private
+   */
+  toggleShoppingTagActive_(id) {
+    const isActive = id === this.pageEl_.id;
+    toggleAttribute(this.shoppingTagEl_, 'active', isActive);
```

Comment:
@gmajoulet should `toggleAttribute()` be within a `mutateElement()`? I figure that there could be dozens of tags in a single story that all have their 'active' attributes toggled every time the user navigates to a different story page, but I'm not 100% sure whether `mutateElement()` is appropriate here

Reply:
Yep you want a `mutateElement` here, as adding a class/attr to the DOM could potentially impact all the CSS layout and make the browser recalculate all of it

Reply:
It has been added 🙌 thank you for catching this and explaining it!